bot.go

```diff
@@ -79,9 +79,37 @@ func (b *Bot) Reply(evt *slack.MessageEvent, text string) {
 	b.RTM.SendMessage(b.RTM.NewOutgoingMessage(text, evt.Channel))
 }
 
+// Reply to a message with Attachments simulate typing
+func (b *Bot) ReplyAttachmentsAndType(evt *slack.MessageEvent, typingDelaySecs int, attachments []slack.Attachment) {
+	b.Type(evt.Channel, "", typingDelaySecs)
```

Comment:
Maybe we should add another function `TypeFor(channel string, typingDelayMS int)` or `TypeForMS` or some other name to allow you to be explicit about the delay.